_stbt/types.py

```diff
@@ -242,13 +247,20 @@ def norm_coords(name_x, name_width, name_right,
 
         return Region(x=x, y=y, right=right, bottom=bottom)
 
-    def translate(self, x=0, y=0):
+    def translate(self, x=None, y=None):
```

Comment:
~I think we could leave the defaults as 0 & 0, for documentation purposes.~ I suppose it's so that you can check if you've passed `y` when you've already given a region.

The docstring needs to explain that you can give it x & y ints or a single `Region`.